compiler/rustc_const_eval/src/interpret/validity.rs

```diff
@@ -36,16 +36,14 @@ macro_rules! throw_validation_failure {
             msg.push_str(", but expected ");
             write!(&mut msg, $($expected_fmt)*).unwrap();
         )?
-        let path = rustc_middle::ty::print::with_no_trimmed_paths!({
```

Comment:
Oh, I hadn't even realized that this was added in the validity checker. Happy to see it gone!